globals/fields.go

```diff
@@ -79,4 +79,7 @@ const (
 	TagsField                            = "tags"
 	CanonicalTagsField                   = "canonical_tags"
 	ConfigurationField                   = "configuration"
+	NodeCredentialsTokenField            = "node_credentials_token"
+	NodeAuthorizationToken               = "node_authorization_token"
```

Comment:
See comment about naming, if it changes don't forget to update here too

Reply:
done!